tools/docker/healthcheck.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 HOST="localhost"
-PORT=6379
+PORT=${PORT:-6379}
```

Comment:
Is the `-` in front of the port intended?

Reply:
Yes, It's the actual syntax? https://stackoverflow.com/a/39296572